app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +157,32 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems.mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
+
+        val drawer = DrawerBuilder(context)
                 .withToolbar(toolbar)
                 .withAccountHeader(accountHeader)
                 .withDrawerItems(generateDrawerItems())
-                .withStickyDrawerItems(generateStickyDrawerItems())
+                .withStickyDrawerItems(stickyDrawerItems)
                 .withOnDrawerItemClickListener { view, id, item ->
                     onDrawerItemClick(view, id, item)
                 }
                 .withShowDrawerOnFirstLaunch(true)
-                .withActionBarDrawerToggleAnimated(true)
                 .withTranslucentStatusBar(true)
+                .withGenerateMiniDrawer(ScreenHelper.isTablet(context))
+                .withSavedInstance(savedInstanceState)
+
+        return if (ScreenHelper.isTablet(context)) drawer.buildView() else drawer.build()
+    }
+
+    private fun buildCrossfader(context: Activity, drawer: Drawer, miniDrawer: MiniDrawer,
+                                savedInstanceState: Bundle?): Crossfader<*>? {
+        return Crossfader<GmailStyleCrossFadeSlidingPaneLayout>()
+                .withContent(context.findViewById(R.id.crossfade_content))
+                .withFirst(drawer.slider, Math.round(UIUtils.convertDpToPixel(300f, context)))
+                .withSecond(miniDrawer.build(context), Math.round(UIUtils.convertDpToPixel(72f, context)))
```

Comment:
The values are from the official crossfader example. They define the width of the first (big) and second (mini) drawer in the crossfader. I don't know if you can get those values from the drawers directly but I will put them in a `const` for now and maybe look for another way.